app/views/elements.nunjucks

```diff
@@ -1,76 +1,78 @@
 {% extends 'layout.nunjucks' %}
 
 {% block content %}
-  <h1>Heading one</h1>
-  <h2>Heading two</h2>
-  <h3>Heading three</h3>
-  <h4>Heading four</h4>
-  <h5>Heading five</h5>
-  <h6>Heading six</h6>
+  <main id="content" role="main" tabindex="-1">
```

Comment:
Should we move this to the main layout view? Seems to be shared across all templates at the moment.